cibyl/plugins/openstack/sources/jenkins_job_builder.py

```diff
@@ -49,6 +51,8 @@ def _parse_xml(self, path):
         result = ""
```

Comment:
This might be a bit of premature optimisation, but what do you think about wrapping the `_parse_xml` in a cache ([docs](https://docs.python.org/3/library/functools.html#functools.cache)), so we don't end up parsing the same file multiple times if multiple arguments are passed, e.g. cibyl query --topology --ip-version

Reply:
np. the trade-off will be an increased memory footprint. 
also the function is moved outside because as a member function it is not cachable.